make.cmd

```diff
@@ -18,6 +18,16 @@ DEL /F /A /Q y.output
 
 golex -o parser/scanner.go parser/scanner.l
 
+@echo [Ast-Parser]
+go get github.com/qiuyesuifeng/goyacc
+go get github.com/qiuyesuifeng/golex
+type nul >>temp.XXXXXX & copy temp.XXXXXX +,,
```

Comment:
I copy [Parser] section to [Ast-Parser] because of same usage from Makefile.

It seems to create a temp file for goyacc, no matter its modified time.

so `type nul >>temp.XXXXXX` is enough to create a new file. 

I will commit it later.